Building.md

```diff
@@ -80,9 +80,10 @@ We make frequent releases, at least weekly. The steps to make a release (say, ve
    clean it up a bit, curating the information for the target audience.
 
  * `git commit -am "Releasing 0.8.$MOC_MINOR"`
- * Create a PR from this commit, and label it `automerge-squash`.  Mergify will
+ * Create a PR from this commit, and label it `automerge-squash`. E.g.
+   with `git push origin HEAD:$USER/0.8.$MOC_MINOR`. Mergify will
    merge it into master without additional approval, within 2 or 3 minutes.
- * `git switch master; git pull`. The release commit should be your `HEAD`
+ * `git switch master; git pull --rebase`. The release commit should be your `HEAD`
```

Comment:
Curious, why the change in process?

Reply:
because we have potentially committed a few revisions on `master` (but pushed to a remote branch), so to avoid conflicts `--rebase` will help (but not guarantee) arriving at ToT. It is harmless basically.

Reply:
Also `git push origin HEAD:$USER/0.8.$MOC_MINOR` will prompt you with an URL that you can visit for creating the PR.

Reply:
This is not a change in the process, just a more precise and detailed writeup.

Reply:
Ok, Thanks!